pkg/config/config_test.go

```
@@ -0,0 +1,644 @@
+	"clustermonitoringerrorbudgetburn",
+	"ClusterProvisioningDelay",
+	"etcddatabasequotalowspace",
+	"insightsoperatordown",
+	"upgradeconfigsyncfailureover4hr",
+	"machinehealthcheckunterminatedshortcircuitsre",
+	"restartcontrolplane",
+	"cannotretrieveupdatessre",
+	"mustgather",
+}
+
+func TestParseConfig(t *testing.T) { //nolint:maintidx // table-driven test with many cases
```

golangci-lint (2.11.4)

[error] cyclomatic complexity 42 of func `TestParseConfig` is high (> 30)

(gocyclo)